### PR TITLE
Added EnumBooleanConverter

### LINKED ISSUE #27: EnumBooleanConverter
```
public class EnumBooleanConverter : IValueConverter
    {
        #region IValueConverter Members
        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
        {
            string parameterString = parameter as string;
            if (parameterString == null)
                return DependencyProperty.UnsetValue;

            if (Enum.IsDefined(value.GetType(), value) == false)
                return DependencyProperty.UnsetValue;

            object parameterValue = Enum.Parse(value.GetType(), parameterString);

            return parameterValue.Equals(value);
        }

        public object ConvertBack(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
        {
            string parameterString = parameter as string;
            if (parameterString == null)
                return DependencyProperty.UnsetValue;

            return Enum.Parse(targetType, parameterString);
        }
        #endregion
    }
```

### PR DESCRIPTION
**Pull request by nazarenomanco**
Closes #27 
Connected to #27

### CODE REVIEW

File: src/net35/Radical.Windows/Presentation/Converters/EnumBooleanConverter.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    [MarkupExtensionReturnType(typeof(EnumBooleanConverter))]
+#endif
+    /// <summary>
+    /// A value converter to convert from enum to bool and back.
+    /// </summary>
+    public class EnumBooleanConverter : IValueConverter
+    {
+        #region IValueConverter Members
+        /// <summary>
+        /// Modifies the source data before passing it to the target for display in the UI.
```
Comment (mauroservienti) [inline src/net35/Radical.Windows/Presentation/Converters/EnumBooleanConverter.cs]: `to display` not `for display`

File: src/net35/Radical.Windows/Presentation/Converters/EnumBooleanConverter.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Data;
+using Topics.Radical.Conversions;
+using System.Windows.Markup;
+
+namespace Topics.Radical.Windows.Presentation.Converters
+{
+#if !SILVERLIGHT
```
Comment (mauroservienti) [inline src/net35/Radical.Windows/Presentation/Converters/EnumBooleanConverter.cs]: Remove the region, Silverligh is not supported anymore.

**Comment by mauroservienti** [comment]
Added a couple of comments, could you please also raise the same issue in the Radical Universal repo? I'll take care of implementing the code there.

**Comment by micdenny** [comment]
I've fixed the description of the PR in `Closes #27` so the issue will be closed automatically when PR is merged, before was `Close issue #27` and it is not in the right format: https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by nazarenomanco** [comment]
@mauroservienti  please review :)
Thanks to @micdenny for correction ;)

**Comment by mauroservienti** [comment]
@micdenny in scenarios such as this one, external contributors, who should be responsible to bump the version number?

**Comment by micdenny** [comment]
> @micdenny in scenarios such as this one, external contributors, who should be responsible to bump the version number?

As team member we have to check that the version is correctly bumped before merge in and we ask the contributors to bump it.

**Comment by mauroservienti** [comment]
@nazarenomanco one last step, can you please bump the version number here: https://github.com/RadicalFx/Radical.Windows/blob/develop/appveyor.yml#L5.

Give that this is a new feature the new version number is `1.4.0-alpha1+{build}`

In order to do that you need to update your fork that seems to be older than the current `develop`

**Comment by mauroservienti** [comment]
this should now be ready to be merged, however I'd like to address https://github.com/RadicalFx/housekeeping/issues/35 before, @micdenny thoughts?

**Comment by mauroservienti** [comment]
Since https://github.com/RadicalFx/housekeeping/issues/35 is more complex than expected I'll merge this one.

**Pull request merged**